templates/content/field--node--oe-sc-person-additional-info.html.twig

```diff
@@ -0,0 +1,18 @@
+{#
+/**
+ * @file
+ * Field template for person additional information.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
```

Comment:
Note: (nothing wrong)
I first thought that we target node type + field name. And then wondered why we don't have `--` between them, only `-`.
But no, the machine name of the field is `oe_sc_person_additional_info`, and it is only used for the `oe_sc_person` ct.
So we don't target this ct explicitly, but it is still ok to just use the field name.